espnet2/diar/espnet_model.py

```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/env python3
```

Comment:
¥Minor comment

Could you remove shebang for not executable scripts?  Shebang is originally used when executing a script, though shebang exists in some scripts now, maybe, to represent python? This might be a style, but I didn't do so in espnet2, so please remove.